src/OverviewPage/OverviewPage.tsx

```diff
@@ -0,0 +1,312 @@
+                                {t('overview.rhosakCallToActionButton')}
+                            </Button>
+                        </CardFooter>
+                    </Card>
+
+                    {/* Database access card */}
+                    <Card>
+                        <CardHeader>
+                            <CardHeaderMain>
+                                <img src={DataServicesLogo} alt="Red Hat OpenShift Data Science logo" style={{ height: '50px' }} />
+                            </CardHeaderMain>
+                        </CardHeader>
+                        <CardTitle>
+                            <Title headingLevel="h2" size="xl">
+                                {t('overview.dbaasTitle')}
+                            </Title>
+                        </CardTitle>
+                        <CardBody>
+                            <Stack hasGutter>
+                                <StackItem className="pf-u-mb-lg">
+                                    <Label>{t('overview.alpha')}</Label>
+                                </StackItem>
+                                <StackItem>{t('overview.dbaasMainText')}</StackItem>
+                                <StackItem className="pf-u-color-200">
+                                    {t('overview.dbaasSecondaryText')}{' '}
+                                    <Button
+                                        data-testid="cardRHODS-linkOpenShift"
+                                        isInline
+                                        variant={ButtonVariant.link}
+                                        component="a"
+                                        target="_blank"
+                                        href="http://openshift.com"
+                                    >
+                                        {' '}
+                                        OpenShift
+                                        <ExternalLinkAltIcon className="pf-u-ml-xs" />
+                                    </Button>
+                                    .
+                                </StackItem>
+                            </Stack>
+                        </CardBody>
+                        <CardFooter>
+                            <Button
+                                data-testid="cardRHODS-buttonCTA"
+                                variant={ButtonVariant.secondary}
+                                component="a"
+                                target="_blank"
+                                href="https://access.redhat.com/documentation/en-us/red_hat_openshift_database_access/1/html-single/quick_start_guide/index"
+                            >
+                                {t('overview.learnMore')} <ExternalLinkAltIcon className="pf-u-ml-sm" />
+                            </Button>
+                        </CardFooter>
+                    </Card>
+                </Grid>
+            </PageSection>
+        </>
+    );
+};
+
+export default OverviewPage;
```

Comment:
Why is this file indented with 4 spaces? The standard is 2. I understand we don't have husky in this repo yet, but please keep an eye open to these details